src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -367,64 +367,71 @@ export const SchedulePage: React.FC = (): JSX.Element => {
                 size="small"
               />
             </Grid>
-            <Grid>
+            <Grid item>
               <Typography variant="h6">{guidLocalDateTime}</Typography>
             </Grid>
           </Grid>
           <Grid item>
-            <Grid container spacing={2} alignItems="center">
-              <Grid item>
-                <DatePicker
-                  variant="inline"
-                  inputVariant="outlined"
-                  format="MMMM d, yyyy"
-                  margin="none"
-                  id="scheduled-start-date"
-                  label="Start date"
-                  value={startDate}
-                  onChange={(date) => {
-                    date && setStartDate(date);
-                  }}
-                  initialFocusedDate={startDate}
-                  disableToolbar
-                  autoOk
-                />
-              </Grid>
-              <Grid item>
-                <DatePicker
-                  variant="inline"
-                  inputVariant="outlined"
-                  format="MMMM d, yyyy"
-                  margin="none"
-                  id="scheduled-end-date"
-                  label="End date"
-                  value={endDate}
-                  onChange={(date) => {
-                    date && setEndDate(date);
-                  }}
-                  initialFocusedDate={endDate}
-                  maxDate={startDate.plus({ days: 30 })}
-                  minDate={startDate.plus({ days: 1 })}
-                  disableToolbar
-                  autoOk
-                />
-              </Grid>
-              <Grid item>
-                <Button
-                  buttonType="positive"
-                  fullWidth
-                  onClick={() => {
-                    executeGetScheduledItemsQuery(
-                      currentScheduledSurfaceGuid,
-                      startDate,
-                      endDate
-                    );
-                  }}
-                >
-                  Get schedule
-                </Button>
+            <LocalizationProvider dateAdapter={AdapterLuxon}>
+              <Grid container spacing={2} alignItems="center">
+                <Grid item sm>
+                  <DatePicker
```

Comment:
`sm` had to be added to the Grid item component here so that the two date fields line up with the button again.

Also, it would be nice to use a `DateRangePicker` instead of two `DatePicker` components here, but both are coming from MUI X, which are still marked as "unstable" in the list of components: https://mui.com/material-ui/discover-more/roadmap/#new-components, so it's a task for another day.